client/landing/site.landing/styl/team.styl

```diff
@@ -12,46 +13,67 @@ team.styl
 */
 }
 
+body.team #kdmaincontainer #main-tab-view
+body.teams #kdmaincontainer #main-tab-view
```

Comment:
Actually it would not because there is no any child style instructions inside of that expression. Here is what I see in the compiled css:
```
...
body.team #kdmaincontainer #main-tab-view,body.teams #kdmaincontainer #main-tab-view{height:auto}.main-header.team{max-width:100%;border-bottom:1px solid rgba(0,0,0,0.15);position:absolute;top:0;height:69px;padding:0 40px;z-index:1;}
...
```
So you can see that after `body.team #kdmaincontainer #main-tab-view,body.teams #kdmaincontainer #main-tab-view{height:auto}` another css instruction takes places. And I can see `body.team #kdmaincontainer #main-tab-view` and `body.teams #kdmaincontainer #main-tab-view` only **once** in css.
I could set a specific css class for element `#main-tab-view` to use easier css construction here but it will require some additional js logic. However, since we already have all css hierarchy built by default and it's enough to override this style, maybe it's better to leave it as is?